@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sensor/fn_targetingFCRUpdate.sqf

```diff
@@ -17,18 +17,21 @@ Author:
     BradMick, Snow(Dryden)
 ---------------------------------------------------------------------------- */
 #include "\fza_ah64_controls\headers\systemConstants.h"
+#include "\fza_ah64_systems\headers\systems.hpp"
 #include "\fza_ah64_mpd\headers\mfdConstants.h"
 params ["_heli"];
 
-private _acBusOn = _heli getVariable "fza_systems_acBusOn";
-private _dcBusOn = _heli getVariable "fza_systems_dcBusOn";
-Private _fcrMode = _heli Getvariable "fza_ah64_fcrMode";
+private _fcrDamage   = _heli getHitPointDamage "hit_msnequip_fcr";
+private _acBusOn     = _heli getVariable "fza_systems_acBusOn";
+private _dcBusOn     = _heli getVariable "fza_systems_dcBusOn";
+Private _fcrMode     = _heli Getvariable "fza_ah64_fcrMode";
 
 if !(_acBusOn && _dcBusOn) exitwith {};
 
 private _fcrTargets = [];
```

Comment:
Could you not `exitWith` before here? Currently it's going to still have to iterate through the loop